cli/src/movement_util.rs

```diff
@@ -129,7 +174,12 @@ fn get_target_commit(
         [target] => target,
         [] => {
             // We found no ancestor/descendant.
-            return Err(user_error(direction.target_not_found_message(args.offset)));
+            return Err(direction.target_not_found_error(
+                workspace_command,
+                args.should_edit,
+                args.offset,
```

Comment:
nit: maybe this is an existing bug, but `args.offset` shouldn't be mentioned if `args.conflict` is set?

Reply:
Yes, it'd be better if this function would take `args.offset` as an optional parameter, since it's always 1 for `prev/next --conflict`.

Reply:
> nit: maybe this is an existing bug, but args.offset shouldn't be mentioned if args.conflict is set?

I'm pretty sure I introduced the bug. Thanks for catching.

>Yes, it'd be better if this function would take args.offset as an optional parameter, since it's always 1 for prev/next --conflict.

I'm now also checking for the conflict arg, and replaced the if/else's with a compound match which is easier to read.